iwyu_tool.py

```diff
@@ -203,6 +209,8 @@ def custom_help():
     parser.add_argument('source', nargs='*',
                         help='Zero or more source files to run IWYU on. '
                         'Defaults to all in compilation database.')
+    parser.add_argument('-i', '--ignore', nargs='+', default=[],
```

Comment:
`+` only requires an argument IF you specify `-i` or `--ignore`. It doesn't make you specify `-i`. While `*` wouldn't be functionally incorrect, it would allow for specifying the ignore flag without specifying any ignore directories, which doesn't make any sense.